source/common/common/logger.h

```diff
@@ -661,7 +661,8 @@ class ExtractedMessage : public spdlog::custom_flag_formatter {
   } while (0)
 
 #define ENVOY_CONN_LOG_EVENT(LEVEL, EVENT_NAME, FORMAT, CONNECTION, ...)                           \
-  ENVOY_LOG_EVENT_TO_LOGGER(ENVOY_LOGGER(), LEVEL, EVENT_NAME, "[C{}] " FORMAT, (CONNECTION).id(), \
+  ENVOY_LOG_EVENT_TO_LOGGER(ENVOY_LOGGER(), LEVEL, EVENT_NAME,                                     \
```

Comment:
I think the previous version was ok and I think it would be better to keep consistent, just need to wrap the ``serializeLogTags`` call with ``fmt::runtime()``, see line 606.

Reply:
It is not enough just for a runtime wrapper, since `:getSink()->logWithStableName` is taking `absl::string_view` and then `fmt::runtime()` is applied again, which I don't want to change the interface right now.

Reply:
Wait, let me take a further look.